modules/architecture-overview.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture.adoc
+
+[id='architecture-overview-{context}']
+= Architecture overview
+
+Like {product-title} v3, {product-title} v4 is a layered system designed to
+expose underlying container images and Kubernetes concepts as accurately as
+possible, with a focus on easy composition of applications by a developer.
+For example, after your cluster is running, install Ruby, push code, and add
+MySQL to the app.
+
+A major difference between {product-title} v3 and v4 is that v4 uses Operators
+as both the fundamental unit of the product and an option for easily deploying
+and managing utilities that your apps use.
+
+{product-title} has an Operator-based architecture of smaller, decoupled units
+that work together. It runs on top of a Kubernetes cluster, with data about the
+objects stored in etcd, a reliable clustered key-value store. Those services are
+broken down by function:
+
+* Operators, which run the core {product-title} services.
```

Comment:
This feels a bit too much like a list - I would hit the three core layers above and give examples of each.  Operators are a detail, really, even if they underlie everything (more glue).